### ISSUE TITLE
What can I expect to happen after I file an issue?

### ISSUE THREAD

**Issue by zcorpan** [webcompat/browser-compat-bugs#7]
The readme could say something about that to motivate the effort of filing an issue.

**Comment by foolip** [comment]
We're figuring this out as we go, but basically the issue should be turned into one or more browser engine bugs, or sometimes spec bugs.

How to communicate back any progress made in the original report I'm not sure.

**Comment by zcorpan** [comment]
A bot that adds a comment in the GH issue when the status of a browser bug is changed to RESOLVED could be useful here. It could be useful for spec PRs also (maybe edit a comment OP to avoid notification DoS).

**Comment by karlcow** [comment]
I think we need to discuss resources and who will be doing triage, verification, and opening the bugs. webcompat.com was created with a spirit of cross-browser participation. On daily basis, it's Mozilla putting in it the work. I would love a better participation model here.

**Comment by guest271314** [comment]
Is this repository still active?

**Comment by karlcow** [comment]
It didn't yield the expected results. 

It's better to directly report on https://webcompat.com/

**Comment by skyschub** [comment]
We should probably archive this repo and add a note to the README forwarding people to webcompat.com. Unless someone shakes their fist at me in disagreement, I'll do so soon. :)

**Comment by karlcow** [comment]
Agreed.

**Comment by guest271314** [comment]
Well, the issues in webcompat.com, including this one I filed https://github.com/webcompat/web-bugs/issues/111283, are lingering unattended to as well.

**Comment by skyschub** [comment]
You filed an issue on a Saturday. It got triaged the following Monday, 18 hours ago. It will be worked on as soon as possible. Please have some patience while the people work through the gigantic pile of issues that come in every week...

**Comment by guest271314** [comment]
> You filed an issue on a Saturday. It got triaged the following Monday, 18 hours ago. It will be worked on as soon as possible. Please have some patience while the people work through the gigantic pile of issues that come in every week...

I have patience. 

I'm just trying to ascertain how committed you folks are to this and what you actually _do_ when bugs are filed.

**Comment by karlcow** [comment]
@guest271314 What @denschub said:
And I encourage you to read https://github.com/webcompat/webcompat.com/blob/main/CODE_OF_CONDUCT.md#our-standards

We all try to work at the best of our abilities and capacities.

**Comment by guest271314** [comment]
I read your COC. Not sure how that is relevant to my question. That doesn't answer the question what you folks _do_ when issues are filed. You appear to have at least a few hundred issues in your queue. If you can't handle to workload I'm willing to help. First I need to know if you contact implementor/vendors, try to fix the bugs yourself, etc. Obviously since you are archiving this repository things are not as active as you initially hoped on the fix it side, quite active on the inbox side. I don't coddle folks that have sensitive needs. What is your mandate?

**Comment by karlcow** [comment]
So the history of this repo. Started from a quick chat in between @foolip and myself, where @foolip was wondering if reporting specific interop bugs for browsers would be more visible here than in https://github.com/webcompat/webcompat.com/issues.

The effort didn't really take off. Probably because there is already a workflow going on. 

If someone reports a bug on https://webcompat.com/ it is being triaged then diagnosed then moved to the appropriate venue. The number of issues are disproportionately firefox, because Firefox nightly/devedition has a menu to make it possible for people do report bugs directly. Some people are reporting WebKit and Blink issues on webcompat.com too, but also on 

* Blink https://bugs.chromium.org/p/chromium/issues/list?q=Hotlist%3DInterop&can=2
* WebKit https://bugs.webkit.org/buglist.cgi?bug_status=UNCONFIRMED&bug_status=NEW&bug_status=ASSIGNED&bug_status=REOPENED&keywords=BrowserCompat%2C%20&keywords_type=allwords&list_id=8605604&query_format=advanced

It's not perfect but we do process the issues here and there. 
@denschub is working for Gecko (Mozilla) 
I'm working for WebKit (Apple)
and @foolip is working for Blink (Google)


About your specific issue on #111283, I'm not sure what is the intent of the issue, but probably better to have the discussion there.

**Comment by guest271314** [comment]
Fair enough. From my experience @foolip is consistent, without an inflated ego or big me little you or I'll ban you personage.

You _are_ the vendors so you have a vested stake in fixing the bugs. The issues filed I noticed are more about specific Web sites than compatibility of specifications and standards implementations. 

The issue I filed reports that Firefox implementation of picture in picture "mode" is not capable of being captured with `getDisplayMedia()` when it should be. There are articles in the Firefox domain that completely fail to mention that.

Best,

/guest271314/
